keyboards/ibm/model_m/ctrl_m/info.json

```diff
@@ -0,0 +1,48 @@
+{
+  "keyboard_name": "ctrl-M replacement controller for the IBM Model M 101/102",
+  "manufacturer": "nuess0r",
+  "url": "https://github.com/qmk/qmk_firmware/tree/master/keyboards/ibm/model_m/ctrl_m",
+  "maintainer": "nuess0r",
+  "usb": {
+    "vid": "0x1D50",
+    "pid": "0x6180",
+    "device_version": "1.0.0",
+    "max_power": 100
+  },
+  "debounce": 5,
+  "matrix_pins": {
+    "cols": ["A15", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "C13", "C14", "C15", "F0", "F1", "A0", "A1", "A2"],
+    "rows": ["A3", "A4", "A5", "A6", "A7", "B0", "B1", "B2"]
+  },
+  "diode_direction": "ROW2COL",
+  "indicators": {
+        "num_lock": "B10",
+        "caps_lock": "B12",
+        "scroll_lock": "B11",
+        "on_state": 0
+  },
+  "ghost": true,
+  "bootmagic": {
+        "enabled": true,
```

Comment:
```suggestion
```

Reply:
Why should I remove this? According to this https://docs.qmk.fm/#/reference_info_json?id=bootmagic the default value is false.

Reply:
Because its then enabled here <https://github.com/qmk/qmk_firmware/pull/22051#discussion_r1328024607>.